### ISSUE TITLE
General Suggestions for data and UI

### ISSUE THREAD

**Issue by chrisaldrich** [dshanske/indieweb-post-kinds#29]
First, I love some of the new changes in v2.2.0!

Here are some additional random thoughts for improving the layout and positioning of some of the final published look of particular posts, if they're not on your "roadmap":

Add an author URL field which could then wrap the data input into "Author/Artist Name" field. For those that want to use it, it makes for a nice way to promote the original author, particularly for properly crediting "bookmarked" items.

In the bookmark post type layout, the order and presentation of the data could be a bit more streamlined. 
`Eg: Bookmarked: (Name/Title) in/from (Site Name/Publication/Album) by (Author/Artist Name)`
Currently it displays as "(Name/Title) on (Site Name/Publication/Album)" and "on" may be an awkward preposition for most cases, though it does work for Albums. Similarly the author name is currently just under the title of the post with no other data, which may be somewhat confusing. I note that some of the post formats are slightly different in how they display, but they could certainly be a bit more systematized/similar with appropriate grammar.

I haven't dug into it, but does the author photo URL have a preset size? One of my sample tests displayed a fairly large photo which was oversize compared to what I might have liked. Otherwise adding a sizing field(s) (height/width or percentage) to add to the image output would be useful for its ultimate display. 

I love the idea of the featured image! I presume it will actually be output in the layout in some future release to make for more visually interesting posts?

**Comment by dshanske** [comment]
I have been thinking about changing the presentation, but I had a lot of trouble with the design. I do want to better customize it instead of it being identical for all post types. 

I like the idea of capturing the author url if they have one. Featured image is currently not widely adopted in microformats but is present in my other source, open graph. 

Re author picture, it does now, but I have had the desire to sideload the image and use Wordpress to load different versions, but it is not as high a priority

**Comment by dshanske** [comment]
Added the URL field, but as I haven't done the design rewrite, it currently doesn't go anywhere. Also have to go back to the parser to see about extracting that info if available. I need a lot more data validation checks also.

I've made rewriting the presentation side the next project. What I did to create the tabs was switch from trying to dynamically generate the view to a series of static tabs. I'm leaning in that direction...making the views individual files and writing one for each type separately that I can edit. The downside is that I'll create more work updating them if I do more changes.
